File: L-31.c

#include "stdio.h"
#pragma warning(disable:4996)
#include "stdlib.h"
#include "string.h"
#include "math.h"
int gg(int h,int g)
{
	float t,j;
	t = (h - 100) * 1.8;
	j = (t - g) * (t-g);
	j = sqrt(j);
	if (j<t*0.1)
	{
		return 0;
	}
	else if (t<g)
	{
		return 1;
	}
	return -1;
}

int main()
{
	int num, h, g;
	scanf("%d",&num);
	for (size_t i = 0; i < num; i++)
	{
		scanf("%d %d", &h, &g);
		int ff = gg(h,g);
		if (ff==0)
		{
			printf("You are wan mei!\n");
		}
		else if (ff==1)
		{
			printf("You are tai pang le!\n");
		}
		else
		{
			printf("You are tai shou le!\n");
		}
	}
	return 0;
}

File: L-31.py

def jj(h,g):
    h-=100
    h*=1.8
    x=(g-h)**2
    x=x**0.5
    if x<h*0.1:
        return 'You are wan mei!'
    elif g>h:
        return 'You are tai pang le!'
    return 'You are tai shou le!'

io=int(input())
for i in range(io):
    x=list(map(int,input().split()))
    print(jj(x[0],x[1]))